### DISCUSSION TITLE
SECURITY ISSUE: Malicious script in https://wnaver.github.io/unsubsc

### DISCUSSION THREAD

**Discussion by Licsak25** [community/community#120423] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

A malicious script was placed at "https://wnaver.github.io/unsubsc" that forwards to a Chinese site.

https://wnaver.github.io/unsubsc?o5RMwuSzuMfEoLet37vHULQJieE0sUDk3VlOeBtV11A%3D

If you follow the link above, you will be directed to a page that is a FORGED Sumitomo-Mitsui Card membership page. This is the link target in the email received as a phishing email.

The entire email, including the envelope, is in the following text file:
[FW 【重要】三井住友銀行カードの身元を確認する1 47 28 PM CDT.txt](https://github.com/community/community/files/15048821/FW.1.47.28.PM.CDT.txt)

decoded body:
[decoded body.txt](https://github.com/community/community/files/15048835/decoded.body.txt)

**Comment by ettaboyle** [comment]
Hi @Licsak25, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
